AMBuildScript

```diff
@@ -96,16 +106,18 @@ class ExtensionConfig(object):
     self.binaries = []
     self.extensions = []
     self.generated_headers = None
-    self.productVersion = None
     self.mms_root = None
     self.sm_root = None
     self.all_targets = []
     self.target_archs = set()
+    self.libsafetyhook = {}
 
     if builder.options.targets:
       target_archs = builder.options.targets.split(',')
     else:
-      target_archs = ['x86', 'x86_64']
+      target_archs = ['x86']
+      if builder.backend != 'amb2':
+        target_archs.append('x86_64')
```

Comment:
This is fine to leave as is. It would be unexpected for anyone not to use amb2. Please undo the change.